lcc_lokavaluto_app_connection/models/account_invoice.py

```diff
@@ -13,8 +13,8 @@ class AccountInvoice(models.Model):
     credit_request_ids = fields.One2many(
         "credit.request", "invoice_id", string="Credit Requests"
     )
-    debit_request_ids = fields.One2many(
-        "debit.request", "debit_move_id", string="Credit Requests"
+    debit_request_ids = fields.Many2many(
+        "debit.request", compute="_compute_debit_request_ids", string="Debit Requests"
```

Comment:
Shouldn't it still stay a One2Many ?

Reply:
If my understanding is good, if `debit_request_ids` is a one2many, this means that there should be a many2one field that link the debit request to the invoice.
the issue is that here we have 2 many2one that must be checked : `debit_move_id` and `commission_move_id` : according to the type of invoice (sale or purchase), we should consider one or the other.